File: _LEGACY_ARCHIVE/MataderoAPI/Models/Cliente.cs
using System;
using System.Collections.Generic;

namespace MataderoAPI.Models
{
    /// <summary>
    /// Modelo de Cliente que recibe los productos despachados
    /// </summary>
    public class Cliente
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        
        public string Nombre { get; set; }
        public string RazonSocial { get; set; }
        public string RUT { get; set; }
        public string Telefono { get; set; }
        public string Email { get; set; }
        
        // Dirección
        public string Direccion { get; set; }
        public string Ciudad { get; set; }
        public string Provincia { get; set; }
        public string CodigoPostal { get; set; }
        
        // Información comercial
        public bool Activo { get; set; } = true;
        public string Contacto { get; set; }
        public string Observaciones { get; set; }
        
        // Relaciones
        public virtual ICollection<Despacho> Despachos { get; set; } = new List<Despacho>();
        
        public DateTime FechaCreacion { get; set; } = DateTime.Now;
        public DateTime? FechaModificacion { get; set; }
        public bool Eliminado { get; set; } = false;
    }
}

File: agronova-api/app/Models/Cliente.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Concerns\HasUuids;

class Cliente extends Model
{
    use HasFactory, HasUuids;

    protected $table = 'Clientes';
    protected $primaryKey = 'Id';

    protected $fillable = [
        'Nombre',
        'RazonSocial',
        'NIT',
        'Telefono',
        'Email',
        'Direccion',
        'Ciudad',
        'Contacto',
        'Observaciones',
        'Activo',
        'Eliminado',
    ];

    protected $casts = [
        'Activo' => 'boolean',
        'Eliminado' => 'boolean',
    ];

    public function despachos()
    {
        return $this->hasMany(Despacho::class, 'ClienteId');
    }
}
